dotnet3.5/S3/CopyObject/CopyObject.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CopyObject]
```

Comment:
Put a blank line between the license statement and the rest of the code. Also, unless another writer is specifically asking for snippet-start and snippet-end tags to be included, I wouldn't worry about adding them.

Reply:
Fixed in all projects.